knox/auth.py

```diff
@@ -62,7 +64,8 @@ def authenticate_credentials(self, token):
         '''
         msg = _('Invalid token.')
         token = token.decode("utf-8")
-        for auth_token in AuthToken.objects.filter(
+        model = self.get_model()
+        for auth_token in model.objects.filter(
```

Comment:
This implies that the new "AuthToken" model should have a field `token_key`, I'm wondering if we should not make this more flexible and have a `get_tokens` method has another way to fetch back the token.

Reply:
Yes sure but still I was challenging the fact that we could make this even more flexible in the case someone needs it. If the point is to make the model pluggable, I would probably go for something fully pluggable unless it's proven too complex.